src/components/SalaryInfo/SalaryInfo.test.jsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from "@testing-library/react";
+import SalaryInfo from "./SalaryInfo";
+import { customRender } from "../../utils/testUtils";
+
+it("Should match the snapshot, with the props", () => {
+  const obj = {
+    Advisor: 25000,
+    Analyst: 5000,
+    Audit: 30000,
+    Consultant: 100000,
+  };
+  const { container } = customRender(
+    <SalaryInfo avgSalaryNumber={"£99,999"} salariesObj={obj} />
```

Comment:
line 13- avgSalaryNumber doesnt exist anymore - you're calculating this inside the component, not passing it as props anymore., so don't need it in any of the renders.

Reply:
Deleted the average salary prop from test